src/conventional/balloon_sonde_json2ioda.py

```diff
@@ -46,8 +46,8 @@
            'relativeHumidity',
            'windEastward',
            'windNorthward']
-obsvars_units = ['K', 'kg kg-1', 'K', 'm s-1', 'm s-1']
-obserrlist = [1.2, 0.75E-3, 1.5, 1.7, 1.7]
+obsvars_units = ['K', '1', 'm s-1', 'm s-1']
+obserrlist = [1.2, 2.0, 1.7, 1.7]
```

Comment:
The idea was 2%/10... I'll bump it up to keep the same units as the observable.

```suggestion
obserrlist = [1.2, 20.0, 1.7, 1.7]
```